Server/docker-compose.yml

```diff
@@ -87,7 +87,7 @@ services:
         condition: service_healthy
       amqp-broker:
         condition: service_healthy
-      directus:
```

Comment:
Why is this replacing the directus condition? We still rely on directus being up in order to start CitrineOS. We should add the logstash condition to the list instead.

Reply:
right, I think that was a misunderstanding - now that you point it out, we need it in COS because we are creating flows right? At that moment, I was thinking that Directus is a separate layer and COS should be able to run without it being up... You are also correct, that yes if we do need Directus to run COS, then I can add that back. The `logstash` is being added below, so adding `directus` back should be all we need...